housekeeping/src/main/java/com/hotels/housekeeping/service/impl/FileSystemHousekeepingService.java

```diff
@@ -76,7 +76,7 @@ private void housekeepPath(LegacyReplicaPath cleanUpPath) {
     final Path rootPath;
     final FileSystem fs;
     try {
-      fs = fileSystemForPath(new Path(cleanUpPath.getPath()));
+      fs = fileSystemForPath(path);
```

Comment:
This was replaced here, not because of a warning, but because I noticed that the same object is created on line 75: `final Path path = new Path(cleanUpPath.getPath());`